tests/Feature/Cities/DeleteCityTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature\Cities;
+
+use Database\Factories\CityFactory;
+use Tests\TestCase;
+
+class DeleteCityTest extends TestCase
```

Comment:
Okay, I was only using it because it was specified on the task assignments (see https://github.com/salva-diaz/travel-agency/pull/7#discussion_r1873926588)